pootle/apps/accounts/managers.py

```diff
@@ -77,3 +81,30 @@ def hide_meta(self):
 
     def meta_users(self):
         return self.get_queryset().filter(username__in=self.META_USERS)
+
+    def get_with_permission(self, permission_code, project, language):
```

Comment:
im not clear what this method gets - could we rename to `get_users_with_permission` if it returns users